src/index.js

```diff
@@ -41,6 +41,21 @@ async function isWithinRange(forwardedFor, fastlyPublicIps, whitelistedIps = '')
     .some((myranges) => (range.isRange ? range.inRange(ip, myranges) : range === ip)));
 }
 
+function getEmbedKind(domainList) {
```

Comment:
nitpick: eventually add some jsdoc of what this does. what is `domainList`? was is the returned value?